x/globalfee/ante/fee.go

```diff
@@ -28,8 +28,8 @@ import (
 var _ sdk.AnteDecorator = FeeDecorator{}
 
 type FeeDecorator struct {
-	GlobalMinFee    globalfee.ParamSource
-	StakingSubspace paramtypes.Subspace
+	GlobalMinFeeSubspace globalfee.ParamSource
```

Comment:
If we're doing a clarity refactor, naming should be consistent with what the variable is imo, `GlobalMinFeeParamSource`